samples/testapp/src/commonMain/kotlin/com/android/identity/testapp/TestAppUtils.kt

```diff
@@ -410,5 +410,4 @@ object TestAppUtils {
             timeValidityEnd
         )
     }
-
-}
\ No newline at end of file
+}
```

Comment:
Since we're not really changing this file anymore it would be good to not add/remove/change whitespace or the imports. Just to avoid creating conflicts for other PRs.